### ISSUE TITLE
WARNING: softmax_cross_entropy_with_logits is deprecated

### ISSUE THREAD

**Issue by DmitriyLanchakov** [philipxjm/Convolutional-Neural-Stock-Market-Technical-Analyser#5]
python 3.6.4
tensorflow 1.6.0

WARNING:tensorflow:From stock_model.py:121: softmax_cross_entropy_with_logits (from tensorflow.python.ops.nn_ops) is deprecated and will be removed in a future version.
Instructions for updating:

Future major versions of TensorFlow will allow gradients to flow
into the labels input on backprop by default.
